### PR TITLE
Feature/menu redux

### PR DESCRIPTION
**Pull request by rdepena**
@tgoc99 @MichaelMCoates @pbaize

### CODE REVIEW

File: js/platform-window.js [hunk 1]
```diff
@@ -1,85 +1,125 @@
...
-            document.getElementById('lock-button').firstChild.src = unlockedSvg;
+            document.getElementById('lock-button').firstChild.src = UNLOCKED_SVG;
         } else {
-            document.getElementById('lock-button').firstChild.src = lockedSvg;
+            document.getElementById('lock-button').firstChild.src = LOCKED_SVG;
         }
     });
     // Before .50 AI version this may throw...
-    fin.Platform.Layout.init({containerId});
+    fin.Platform.Layout.init({CONTAINER_ID});
```
Comment (pbaize) [inline js/platform-window.js:17]: Does the api expect this casing?

File: app.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "runtime": {
-        "arguments": "--v=1 --inspect",
-        "version": "canary"
+        "arguments": "",
+        "version": "alpha"
```
Comment (tgoc99) [inline app.json:4]: should we keep on canary? Its a good source for finding out issues / regressions...

Comment (rdepena) [reply app.json:4]: Asking Nicholas as it might be disruptive for customers as we will break stuff.

File: js/platform-window.js (hunk 1 above)
Comment (rdepena) [reply js/platform-window.js]: Guess it only worked because I did not remove the id from the element.

File: app.json (hunk 2 above)
Comment (MichaelMCoates) [reply app.json:4]: Should we make a develop branch?

Comment (MichaelMCoates) [reply app.json:4]: Also we'll run into this with styling changes, because all my styles would land in canary.

File: styles/frame-styles.css [hunk 3]
```diff
@@ -4,6 +4,7 @@
     --frame-background: #121212;
     --title-bar: #000000;
     --tab-text: #FFFFFF;
+    --body-background-color: #b8b8b8;
```
Comment (MichaelMCoates) [inline styles/frame-styles.css:7]: So this is called `body-background-color` but I see it used as `color` instead, which sets font color. Not sure if that was your intention

Comment (rdepena) [reply styles/frame-styles.css:7]: probably not.

File: app.json (hunk 2 above)
Comment (rdepena) [reply app.json]: Made a dev branch: https://github.com/openfin/platform-api-project-seed/pull/13

File: styles/frame-styles.css (hunk 3 above)
Comment (rdepena) [reply styles/frame-styles.css]: Fixed, but moved the PR to a new dev branch.

**Pull request closed without merging**